packages/app-admin-ui/client/pages/Item/Footer.js

```diff
@@ -75,38 +77,40 @@ function Reset({ canReset, onReset }) {
   );
 }
 
-export default memo(function Footer(props) {
-  const { onSave, onDelete, canReset, updateInProgress, onReset, hasWarnings, hasErrors } = props;
-  const cypressId = 'item-page-save-button';
+export default memo(
+  ({ onSave, onDelete, canReset, updateInProgress, onReset, hasWarnings, hasErrors }) => {
+    const { list } = useList();
+    const cypressId = 'item-page-save-button';
 
-  return (
-    <Fragment>
-      <Toolbar key="footer">
-        <div css={{ display: 'flex', alignItems: 'center' }}>
-          <LoadingButton
-            appearance={hasWarnings && !hasErrors ? 'warning' : 'primary'}
-            id={cypressId}
-            isDisabled={updateInProgress || hasErrors}
-            isLoading={updateInProgress}
-            onClick={onSave}
-            style={{ marginRight: 8 }}
-            type="submit"
-          >
-            {hasWarnings && !hasErrors ? 'Ignore Warnings and Save Changes' : 'Save Changes'}
-          </LoadingButton>
-          <Reset canReset={canReset} onReset={onReset} />
-        </div>
-        <div>
-          <Button
-            appearance="danger"
-            isDisabled={updateInProgress}
-            variant="nuance"
-            onClick={onDelete}
-          >
-            Delete
-          </Button>
-        </div>
-      </Toolbar>
-    </Fragment>
-  );
-});
+    return (
+      <Fragment>
+        <Toolbar key="footer">
+          <div css={{ display: 'flex', alignItems: 'center' }}>
+            <LoadingButton
+              appearance={hasWarnings && !hasErrors ? 'warning' : 'primary'}
+              id={cypressId}
+              isDisabled={updateInProgress || hasErrors || !list.access.update}
+              isLoading={updateInProgress}
+              onClick={onSave}
+              style={{ marginRight: 8 }}
+              type="submit"
+            >
+              {hasWarnings && !hasErrors ? 'Ignore Warnings and Save Changes' : 'Save Changes'}
+            </LoadingButton>
+            <Reset canReset={canReset && !list.access.update} onReset={onReset} />
```

Comment:
This logic should probably be moved to `ItemDetails` where the rest of the `canReset` value is calculated. Having this here splits the logic for that value across two different files which makes it very difficult to reason about.

Reply:
done